tests/Util/PHPVersionTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Raven\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+use Raven\Util\PHPVersion;
+
+class PHPVersionTest extends TestCase
+{
+    /**
+     * @dataProvider versionProvider
+     * @param $expected
```

Comment:
I would remove these lines as they are not useful at all (they do not specify a typehint nor a description of the parameter) and we don't usually add docblocks to the methods of the unit test classes